guides/common/modules/proc_configuring-an-external-dhcp-server.adoc

```diff
@@ -122,15 +122,15 @@ _990_
 ifndef::foreman-deb[]
 ----
 # {package-install} nfs-utils
-# systemctl enable rpcbind nfs-server
-# systemctl start rpcbind nfs-server nfs-lock nfs-idmapd
+# systemctl enable --now rpcbind nfs-server
+# systemctl start nfs-lock nfs-idmapd
```

Comment:
what's `nfs-lock`, neither my EL8 nor my F36 systems know such a service?!

ah, looking at an EL7 system:
```console
# systemctl status nfs-lock
● rpc-statd.service - NFS status monitor for NFSv2/3 locking.
   Loaded: loaded (/usr/lib/systemd/system/rpc-statd.service; static; vendor preset: disabled)
   Active: inactive (dead)
# ls -alh /usr/lib/systemd/system/nfs-lock.service 
lrwxrwxrwx. 1 root root 17 Dec 21  2020 /usr/lib/systemd/system/nfs-lock.service -> rpc-statd.service
```

So I guess this should be swapped out, and also be moved to `enable --now`?

Can we then just fold the two code sections into one, only differentiating on the package line?

Reply:
It looks like `nfs-server` has wants on `rpcbind` and `nfs-idmapd` so they will be started. Should we reduce the list of services to just `nfs-server`? If you only want an NFSv4 server that's really all you need according to https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_file_systems/configuring-an-nfsv4-only-server_managing-file-systems